streamly.cabal

```diff
@@ -391,13 +547,13 @@ library
                      , Streamly.Internal.Data.Stream.StreamD.Type
                      , Streamly.Internal.Data.Stream.StreamDK.Type
                      , Streamly.Internal.Data.Unfold.Type
-                     , Streamly.Internal.Data.Producer.Type
-                     , Streamly.Internal.Data.Producer
-                     , Streamly.Internal.Data.Producer.Source
-                     , Streamly.Internal.Data.Sink.Type
-                     , Streamly.Internal.Data.Parser.ParserK.Type
-                     , Streamly.Internal.Data.Parser.ParserD.Type
-                     , Streamly.Internal.Data.Pipe.Type
+                     , streamly-core:Streamly.Internal.Data.Producer.Type
+                     , streamly-core:Streamly.Internal.Data.Producer
+                     , streamly-core:Streamly.Internal.Data.Producer.Source
+                     , streamly-core:Streamly.Internal.Data.Sink.Type
+                     , streamly-core:Streamly.Internal.Data.Parser.ParserK.Type
+                     , streamly-core:Streamly.Internal.Data.Parser.ParserD.Type
+                     , streamly-core:Streamly.Internal.Data.Pipe.Type
```

Comment:
Why only these modules have a `streamly-core:` prefix, and not others?

Reply:
Ah, they should be removed. I forgot to remove all of them.